dist/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script><script defer src="print.bundle.js"></script></head>
+<body>
+
+    <div id="todo">
+        <div>
+            <h1 id="todo-title">Today's To Do List</h1>
+        </div>
+        <ul id="todo-list">
+        
+        </ul>
+        <button id="clear-button">Clear all Completed</button>
```

Comment:
The display of the tasks looks quite good. Kindly include an input element for adding tasks as shown on the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/215184503-37e05d57-2cb8-4f4d-bd52-c102f6368ae0.png)

Design guide:
![image](https://user-images.githubusercontent.com/20554583/215184524-4e72cd71-3833-4400-a28d-9473fcea67ec.png)